trading_ig/rest.py

```diff
@@ -76,6 +76,11 @@ def _create_first(self, endpoint, params, session):
         response = session.post(url,
                                 data=json.dumps(params),
                                 headers=self.HEADERS['BASIC'])
+        if response.ok:
```

Comment:
Why not using:

```python
    if not response.ok:
        ...
```

Reply:
Just the way I was thinking ! - I will make an update.